Externals/expr/include/expr.h

```diff
@@ -934,6 +978,8 @@ extern "C" {
         free(e->param.func.context);
       }
     }
+    else if (e->type == OP_STRING)
+      free(e->param.str.s);
```

Comment:
Nit: Please use braces here. Leaving them out clashes with the rest of the expr library and also with Dolphin's own style guide.